lib/embulk/input/google_analytics/plugin.rb

```diff
@@ -95,7 +95,7 @@ def init
         end
 
         def run
-          client = Client.new(task, preview?)
+          client = Client.new(task)
```

Comment:
I see for the previewing issue, `No input records to preview` occurring.
But this fix seems also breaks the `embulk preview` command. `embulk preview` try to fetch all of records as possible with this change, same as `embulk run`, isn't it?